clients/cli/src/clap_app.rs

```diff
@@ -1721,6 +1722,47 @@ pub fn app<'a>(
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UnwrapLamports.into())
+                .about("Unwrap lamports from a SOL token account")
+                .arg(
+                    Arg::with_name("amount")
+                        .value_parser(Amount::parse)
+                        .value_name("TOKEN_AMOUNT")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("Amount to unwrap, in tokens; accepts keyword ALL"),
+                )
+                .arg(
+                    Arg::with_name("recipient")
+                        .validator(|s| is_valid_pubkey(s))
+                        .value_name("RECIPIENT_ACCOUNT_ADDRESS")
+                        .takes_value(true)
+                        .index(2)
+                        .help("Specify the address to recieve the unwrapped SOL. \
+                               Defaults to the owner address.")
```

Comment:
Let's avoid a default
```suggestion
                        .help("Specify the address to receive the unwrapped SOL.")
```